tray/src/com/dvt/main/creational/abstract_factory/Client.java

```diff
@@ -0,0 +1,37 @@
+<<<<<<< HEAD
+package com.dvt.creational.abstract_factory;
+
+import com.dvt.creational.abstract_factory.factory.FactoryProducer;
+import com.dvt.creational.abstract_factory.factory.StampingEquipmentFactory;
+=======
+package creational.abstract_factory;
+
+import creational.abstract_factory.factory.FactoryProducer;
+import creational.abstract_factory.factory.StampingEquipmentFactory;
+>>>>>>> main
```

Comment:
Please fix merge conflict.
```suggestion
<<<<<<< HEAD
package com.dvt.creational.abstract_factory;

import com.dvt.creational.abstract_factory.factory.FactoryProducer;
import com.dvt.creational.abstract_factory.factory.StampingEquipmentFactory;
=======
package creational.abstract_factory;

import creational.abstract_factory.factory.FactoryProducer;
import creational.abstract_factory.factory.StampingEquipmentFactory;
>>>>>>> main
```